dev/tests/integration/framework/Magento/TestFramework/Annotation/ConfigFixture.php

```diff
@@ -67,12 +68,12 @@ protected function _getConfigValue($configPath, $scopeCode = null)
      *
      * @param string $configPath
      * @param string $value
-     * @param string|bool|null $storeCode
+     * @param string|bool|null $scopeCode
      */
-    protected function _setConfigValue($configPath, $value, $storeCode = false)
+    protected function _setConfigValue($configPath, $value, $scopeCode = false, $scopeType = ScopeInterface::SCOPE_STORE)
```

Comment:
Adding parameters in protected methods is not allowed see our 
[**Backward Compatible Guide**](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)